FirebaseFunctions.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
```

Comment:
Looks like the `quickstart / functions` CI is failing: https://github.com/firebase/firebase-ios-sdk/actions/runs/3109715595/jobs/5040191907

It looks due to FirebaseUI neededing to be updated: https://github.com/firebase/FirebaseUI-iOS/blob/master/FirebaseAuthUI.podspec#L21

@morganchen12, is there a simple fix for this or does it require a FirebaseUI release?

Reply:
cc: @paulb777

Reply:
Discussed offline. Will do a FirebaseUI release to relax the version requirements.